js/data/locale/mt/unitfmt.json

```diff
@@ -8,10 +8,10 @@
             "degree": "#{n}°",
             "arc-minute": "#{n}′",
             "arc-second": "#{n}″",
-            "square-kilometer": "#{n} km²",
+            "square-kilometer": "#{n} kilometru kwadru",
```

Comment:
These need to be Maltese plurals. They should have different cases for the different plural categories.

Reply:
Yes, the problem is, Maltese have several plural categories. one, few, many, other.
so the format data form would have like this,
`one#{n} |few#{n} |many#{n} |#{n} `

 but I'm don't get how to get 'few` category translation correctly. so I'll just update the same with many
`one#{n}kilometru kwadru |few#{n} kilometri kwadri|many#{n}kilometri kwadri |#{n}kilometri kwadri `

square kilometer: kilometru kwadru
square killometers :  kilometri kwadri

Reply:
Enter the following into google:

1 square kilometer = Kilometru kwadru
2 square kilometers = 2 kilometri kwadri
14 square kilometers = 14-il kilometru kwadru
21 square kilometers = 21 kilometru kwadru

This translates to the following plural string:

"one#Kilometru kwadru|few#{n} kilometri kwadri|many#{n}-il kilometru kwadru|other#{n} kilometru kwadru"